network/p2p/dns/resolver_test.go

```diff
@@ -23,9 +25,13 @@ const happyPath = true
 // instead of going through the underlying basic resolver, and hence through the network.
 func TestResolver_HappyPath(t *testing.T) {
 	basicResolver := mocknetwork.BasicResolver{}
-	resolver := NewResolver(metrics.NewNoopCollector(), WithBasicResolver(&basicResolver))
+	resolver := NewResolver(zerolog.Nop(), metrics.NewNoopCollector(), WithBasicResolver(&basicResolver))
```

Comment:
Please use `unittest.Logger`. We use logs to investigate failure/flakiness on CI.

Reply:
946c4ae

Reply:
67f581e